plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/snowflake/TestingSnowflakeServer.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.iceberg.catalog.snowflake;
+
+import io.airlift.log.Logger;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Optional;
+import java.util.Properties;
+import java.util.function.Function;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static io.trino.testing.TestingProperties.requiredNonEmptySystemProperty;
+
+public class TestingSnowflakeServer
+{
+    public enum TableType
+    {
+        NATIVE, ICEBERG
+    }
+
+    private static final Logger LOG = Logger.get(TestingSnowflakeServer.class);
+
+    public static final String SNOWFLAKE_JDBC_URI = requiredNonEmptySystemProperty("testing.snowflake.catalog.account-url");
```

Comment:
`SNOWFLAKE_JDBC_URI` -> `SNOWFLAKE_JDBC_URL`

Reply:
Keeping as is as Iceberg Snowflake catalog refers it as URI.